packages/search-in-workspace/src/browser/search-in-workspace-result-tree-widget.tsx

```diff
@@ -759,8 +764,8 @@ export class SearchInWorkspaceResultTreeWidget extends TreeWidget {
                     },
                     options: {
                         overviewRuler: {
-                            color: 'rgba(230, 0, 0, 1)',
```

Comment:
have you tried 'var(--theia-editor-findMatchHighlightBackground)'? coding guidelines explain how css variable name is derived from variable name: https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#theming-no-css-color-values

Reply:
Yes, unfortunately, it does not work to use:
```ts
'var(--theia-editor-findMatchHighlightBackground)'
```

Reply:
ok